Cargo.lock

```diff
@@ -2892,7 +2892,7 @@ dependencies = [
  "serde",
  "serde_json",
  "structopt",
- "symbolic-common",
+ "symbolic-common 10.2.1",
```

Comment:
pls also bump the version in the `process-event` utility :-)

Reply:
done